.github/workflows/update_chains_preconfigured.yaml

```
@@ -0,0 +1,62 @@
+name: Update chains preconfigured
+
+on:
+  schedule:
+    - cron: '0 8 1/3 * *'
+  workflow_dispatch:
+  pull_request:
+
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-preconfigured:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout current repository to Master branch
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 🛠 Set up actual paths
+        uses: ./.github/workflows/setup-path
+
+      - name: ⚙️ Install dependencies
+        run: make init
+
+      - name: 🦾 Update files and data in chains preconfigured
+        run: make update-chains-preconfigured
+
+      - name: Make Pull Request
+        uses: ./.github/workflows/make-pull-request
+        with:
+          commit-files: ${{ env.COMMIT_FILE }}
```

would be nice to use all changes in chains directory
```suggestion
          commit-files: chains/**
```